src/Sylius/Component/Core/Context/ProcessContextInterface.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Sylius\Component\Core\Context;
+
+interface ProcessContextInterface
+{
+    public const CLI = 'cli';
```

Comment:
Is it worth it to have separate interface for this const? Can't we place it in `CommandBasedContextChecker`? Also, this const could be private